static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -80,6 +133,10 @@
     }
 }
 
+.content__wrapper--standfirst--immersive-article {
```

Comment:
Standfirst is supposed to be the modifier. We have `content__wrapper` and `content__wrapper--headline` already. But I guess I don't have to be this specific. I changed it to `content__wrapper--standfirst`. We were already using that class for immersive articles, but just for immersive articles.